modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -73,6 +75,14 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
     assertShadowHostState(editor, shouldExist);
   };
 
+  const keystokes = (editor: Editor) => {
```

Comment:
This isn't a very descriptive function name. Can we please rename this to what it's doing? For example `openDialogViaShortcut`, `openDialogWithKeyboard` or similar.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/b30f7ae60728c967a43223fb42233939f24e39e4